Other/MergeShops/Adam1a1Merge.cs

```diff
@@ -0,0 +1,152 @@
+using System.ComponentModel.Design;
+using System.Diagnostics.Contracts;
+using System.Runtime.ExceptionServices;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System.Reflection.PortableExecutable;
```

Comment:
no usings above the includes, also there were more than likely generated when you were doing stuff below (comes with auto-using Extention and can happen needs fixed manually) 
++ none of these above the includes are needed 👍